packages/amplify_auth_cognito/lib/src/CognitoPasswords/CognitoResetPasswordResult.dart

```diff
@@ -15,6 +15,8 @@
 
 import 'package:amplify_auth_plugin_interface/amplify_auth_plugin_interface.dart';
 
+/// Cognito extension of ResetPasswordResult
 class CognitoResetPasswordResult extends ResetPasswordResult {
+  /// Default constructor for CognitoResetPasswordResult
   CognitoResetPasswordResult({nextStep, isPasswordReset}) : super(isPasswordReset: isPasswordReset, nextStep: nextStep);
```

Comment:
I don't think there is an `@inherit` annotation to automatically copy the super class's doc. If not, we should just copy paste it here. When customers are using `CognitoResetPasswordResult`, they won't get any help what that is until they reach `ResetPasswordResult` e.g. in IDE.

Reply:
For this particular example, I'm doing this:
```
/// Wraps the result of a Cognito reset password operation,
/// which will include a [isPasswordReset] flag and a [nextStep] detailing the 
/// next step in the password reset process (if any). 
```
This follows the do's and dont's of flutter documentation, which suggests description of attributes and/or parameters with square brackets rather than "@param" style documentation (I'm assuming this preference has something to do with the lack of overloads and a decreased need to itemize parameters.